kotlin/internal/jvm/jvm.bzl

```diff
@@ -513,13 +513,13 @@ kt_compiler_plugin = rule(
             default = False,
         ),
         "_jetbrains_deshade_rules": attr.label(
-            default = "@io_bazel_rules_kotlin//kotlin/internal/jvm:jetbrains-deshade.jarjar",
+            default = "//kotlin/internal/jvm:jetbrains-deshade.jarjar",
```

Comment:
Sometimes here you're using a plain string `"//third_party/foo"`, sometimes a `Label("//third_party/foo")`.  Should we be consistent, or is this a place where that's actually advisable or necessary?

Reply:
Good catch. We don't use compiler plugins in base, so I missed it.

We should *always use Label*. Label provides the fully qualified target, where as string is resolved relatively against the main repo.